harness-requirements.typ

```diff
@@ -11,12 +11,22 @@ Requirement priority is in ascending order, with P1 being the highest priority a
 #let couldHave = [Could Have]
 #let wouldBeNiceToHave = [Would Be Nice To Have]
 
+#let VERY_HIGH = 1
+#let HIGH = 2
+#let MEDIUM = 3
+#let LOW = 4
+#let VERY_LOW = 5
+
 #let priority(pri) = {
-	[P] + str(pri)
+	if(pri == VERY_HIGH) { rgb("#FF0407") }
+	else if(pri == HIGH) { rgb("#ff390a") }
+	else if(pri == MEDIUM) { rgb("#ecff09") }
+	else if(pri == LOW) { rgb("#1ae53e") }
+	else if(pri == VERY_LOW) { rgb("#1800FF") }
 };
 
 #let req(text, necessity, pri) = {
-	priority(pri) + [ ] + necessity + [ ] + text
+	"-" + necessity + [ ] + text + [ ] + square(size: 5pt, fill: priority(pri)) + [ ] + "\n"
 };
```

Comment:
The priority number is not rendered anymore. It just shows a color.

The `priority` function should do everything required to render the priority number, colors, shapes, whatever.